exonum/src/blockchain/transaction.rs

```diff
@@ -37,26 +37,33 @@ const TRANSACTION_STATUS_OK: u16 = MAX_ERROR_CODE + 1;
 // `Err(TransactionErrorType::Panic)`.
 const TRANSACTION_STATUS_PANIC: u16 = TRANSACTION_STATUS_OK + 1;
 
-/// Return value of the `Transaction`'s `execute' method. Changes made by the transaction are
-/// discarded if `Err` is returned, see `Transaction` documentation for the details.
+/// Returns the result of the `Transaction` `execute` method.
+///
+/// This result may be
+/// either an empty `unit` type, in case of success, or an `ExecutionError`, if execution has
```

Comment:
Nitpick: I wouldn't format "unit [type]" as code; it's not a literal name of the type in Rust (which is `()`).